### ISSUE TITLE
Page Previews Loading 5.5 Data

### ISSUE THREAD

**Issue by MlleCanary** [The-Balance-FFXIV/glam#309]
![image](https://user-images.githubusercontent.com/88956860/204967806-f424a396-c56d-4ea0-937d-eb3ae6dba01e.png)
 Image provided.

**Comment by nonowazu** [comment]
This is because the opengraph description is summarizing the body content:

```
king@xenogears:~$ wget -q -O - https://www.thebalanceffxiv.com/jobs/healers/scholar/best-in-slot/ | grep 'og:desc'
<meta property="og:description" content="SCH BiS and Gear Options 5.55 Savage Sets So, why the hell is SCH BiS 2.32 GCD this tier? The short answer is that our gear absolutely sucks for 2.41 this tier. We either have to take on a bunch of piety or use crafted gear, and neither one is efficient. 2.32 GCD lines up AF well, but sits on a sort of halfway point of DoT timings that requires us to alternate between 12 GCD and 13 GCD Bios, which can result in lost Bio ticks.">
```

If you look at the `body` of that [page](https://github.com/The-Balance-FFXIV/balance-static/blob/a03f5a2279d7884d078210dd8b6c5d2e6d39a5d4/content/jobs/healers/scholar/best-in-slot.md):

```md
# SCH BiS and Gear Options

# 5.55 Savage Sets

***So, why the hell is SCH BiS 2.32 GCD this tier?***  \
The short answer is that our gear absolutely sucks for 2.41 this tier. We either have to take on a bunch of piety or use crafted gear, and neither one is efficient. 2.32 GCD lines up AF well, but sits on a sort of halfway point of DoT timings that requires us to alternate between 12 GCD and 13 GCD Bios, which can result in lost Bio ticks. This still makes it the most efficient GCD that we're able to hit at the moment damage wise, that doesn't throw off AF alignment or give us meme levels of mana efficiency.
```

Whereas the rendering for that template happens entirely out of the `bis` field in the front matter header. If you go to that link and click `blame`, you'll see that the content there was put in place 15 months ago — and the summary is drawing from that. The [current](https://github.com/The-Balance-FFXIV/balance-static/blob/a03f5a2279d7884d078210dd8b6c5d2e6d39a5d4/static/admin/config.yml#L2324) editor config doesn't allow anyone to edit the body field, so that has remained untouched for the last ~15 months.

tl;dr - this isn't a bug in glam or the editor; remove the stuff in the body and it'll fix itself. For actual opengraph embeds, #308 has seo embeds which should fix that.

**Comment by nonowazu** [comment]
I think this issue can be closed. I spot-checked a few `best-in-slot.md` files and didn't see a body anymore.
